components/homme/cmake/machineFiles/aurora-aot.cmake

```diff
@@ -57,8 +50,6 @@ set (ENABLE_HORIZ_OPENMP OFF CACHE BOOL "")
 
 set (HOMME_TESTING_PROFILE "dev" CACHE STRING "")
 
-set (USE_NUM_PROCS 4 CACHE STRING "")
-
-SET (USE_MPI_OPTIONS "--bind-to core" CACHE FILEPATH "")
-
+set (USE_NUM_PROCS 12 CACHE STRING "")
 
+SET (USE_MPI_OPTIONS "--pmi=pmix --cpu-bind list:0-7,104-111:8-15,112-119:16-23,120-127:24-31,128-135:32-39,136-143:40-47,144-151:52-59,156-163:60-67,164-171:68-75,172-179:76-83,180-187:84-91,188-195:92-99,196-203 gpu_tile_compact.sh" CACHE FILEPATH "")
```

Comment:
is tile script always in a user's path?

Reply:
Yes, the tile binding script is always in the SYSTEM path for the users